src/client/mod.rs

```diff
@@ -827,7 +827,7 @@ impl KafkaClient {
             for tp in resp.topic_partitions {
                 let e = res.entry(tp.topic).or_insert(vec![]);
                 for p in tp.partitions {
-                    e.push(p.into_offset());
+                    e.push(try!(p.into_offset()));
```

Comment:
feel free to get it right. and take the time it requires. there's certainly no pressure to get this out as soon as possible. fortunately, this is just for fun :)